cmake/Toolchains/arm_cortex_a72_hardfp_native.cmake

```diff
@@ -11,7 +11,16 @@
 # Usage: cmake -DCMAKE_TOOLCHAIN_FILE=<this file> <source directory>
 ########################################################################
 set(CMAKE_CXX_COMPILER g++)
-set(CMAKE_C_COMPILER  gcc)
-set(CMAKE_CXX_FLAGS "-march=armv8-a -mtune=cortex-a72 -mfpu=neon-fp-armv8 -mfloat-abi=hard" CACHE STRING "" FORCE)
-set(CMAKE_C_FLAGS ${CMAKE_CXX_FLAGS} CACHE STRING "" FORCE) #same flags for C sources
-set(CMAKE_ASM_FLAGS "${CMAKE_CXX_FLAGS} -mthumb -g" CACHE STRING "" FORCE) #same flags for asm sources
+set(CMAKE_C_COMPILER gcc)
+set(
+  CMAKE_CXX_FLAGS
+  "-march=armv8-a -mtune=cortex-a72 -mfpu=neon-fp-armv8 -mfloat-abi=hard"
+  CACHE STRING "" FORCE)
+set(
+  CMAKE_C_FLAGS
+  ${CMAKE_CXX_FLAGS}
+  CACHE STRING "" FORCE) #same flags for C sources
```

Comment:
Not sure, but this looks pretty different than GR's formatting. You're closing an issue that says "format like GNU Radio" with this!